tests/fixtures/ecma-version/13/regexp-match-indices/regexp-match-indices.result.js

```diff
@@ -0,0 +1,195 @@
+export default {
+    "type": "Program",
+    "loc": {
+        "start": {
+            "line": 1,
+            "column": 0
+        },
+        "end": {
+            "line": 1,
+            "column": 26
+        }
+    },
+    "range": [
+        0,
+        26
+    ],
+    "body": [
+        {
+            "type": "VariableDeclaration",
+            "loc": {
+                "start": {
+                    "line": 1,
+                    "column": 0
+                },
+                "end": {
+                    "line": 1,
+                    "column": 26
+                }
+            },
+            "range": [
+                0,
+                26
+            ],
+            "declarations": [
+                {
+                    "type": "VariableDeclarator",
+                    "loc": {
+                        "start": {
+                            "line": 1,
+                            "column": 6
+                        },
+                        "end": {
+                            "line": 1,
+                            "column": 25
+                        }
+                    },
+                    "range": [
+                        6,
+                        25
+                    ],
+                    "id": {
+                        "type": "Identifier",
+                        "loc": {
+                            "start": {
+                                "line": 1,
+                                "column": 6
+                            },
+                            "end": {
+                                "line": 1,
+                                "column": 9
+                            }
+                        },
+                        "range": [
+                            6,
+                            9
+                        ],
+                        "name": "re1"
+                    },
+                    "init": {
+                        "type": "Literal",
+                        "loc": {
+                            "start": {
+                                "line": 1,
+                                "column": 12
+                            },
+                            "end": {
+                                "line": 1,
+                                "column": 25
+                            }
+                        },
+                        "range": [
+                            12,
+                            25
+                        ],
+                        // Only supported in Node.js 16+
+                        "value": /^(12|14)\./.test(process.versions.node) ? null : eval("a+(?<Z>z)?/d"),
```

Comment:
Node.js 12 and 14 don't yet support the `d` flag while Node.js 16 does, which leads to different results here depending on the version of Node.js being used.

Reply:
there is already a function to handle it: https://github.com/eslint/espree/blob/c8ca49a281eb74bb81e9bc3a60300011e8b29e8d/tests/lib/conditional-regex-value.js#L2-L12

in this way, no need to hard code Node.js version, and `eval()`

Reply:
Nice! I’ll update to use that.